diff --git a/web/core/modules/content_moderation/src/Form/EntityModerationForm.php b/web/core/modules/content_moderation/src/Form/EntityModerationForm.php
index 919f4e5b..85457807 100644
--- a/web/core/modules/content_moderation/src/Form/EntityModerationForm.php
+++ b/web/core/modules/content_moderation/src/Form/EntityModerationForm.php
@@ -102,7 +102,7 @@ public function buildForm(array $form, FormStateInterface $form_state, ContentEn
     if ($current_state) {
       $form['current'] = [
         '#type' => 'item',
-        '#title' => $this->t('Moderation state'),
+        '#title' => $this->t('Current state'),
         '#markup' => $workflow->getTypePlugin()->getState($current_state)->label(),
       ];
     }
@@ -116,12 +116,6 @@ public function buildForm(array $form, FormStateInterface $form_state, ContentEn
       '#options' => $target_states,
     ];
 
-    $form['revision_log'] = [
-      '#type' => 'textfield',
-      '#title' => $this->t('Log message'),
-      '#size' => 30,
-    ];
-
     $form['submit'] = [
       '#type' => 'submit',
       '#value' => $this->t('Apply'),
